# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/src/media-conduit/MediaConduitInterface.h
# Commit: 941a85e51309
# Full Hash: 941a85e51309894ce29857b8b30578856ee99921
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2018-11-09 10:01:40
# Regressor Bug: 1376873
# File Overlap Count: 2
# Description:
#   Bug 1376873 - Add RtpPacketQueue; r=pehrsons
#   
#   We'll need to queue rtp packets in both the AudioConduit and the VideoConduit.
#   This adds a class to manage the packet queue to reduce duplicated code between
#   the conduits.
# ==============================================================================

diff -r 950e95f79312 -r 941a85e51309 media/webrtc/signaling/src/media-conduit/MediaConduitInterface.h
--- a/media/webrtc/signaling/src/media-conduit/MediaConduitInterface.h	Tue Feb 20 15:23:09 2018 -0500
+++ b/media/webrtc/signaling/src/media-conduit/MediaConduitInterface.h	Thu Mar 22 09:44:49 2018 -0400
@@ -255,6 +255,8 @@
 
   virtual void SetPCHandle(const std::string& aPCHandle) = 0;
 
+  virtual MediaConduitErrorCode DeliverPacket(const void *data, int len) = 0;
+
   virtual void DeleteStreams() = 0;
 
   NS_INLINE_DECL_THREADSAFE_REFCOUNTING(MediaSessionConduit)